apm-agent-plugins/apm-api-plugin/src/test/java/co/elastic/apm/api/AbstractSpanImplAccessor.java

```diff
@@ -20,7 +20,7 @@
 
 public class AbstractSpanImplAccessor {
 
-    public static co.elastic.apm.agent.impl.transaction.Transaction accessTransaction(Transaction t) {
-        return (co.elastic.apm.agent.impl.transaction.Transaction) ((TransactionImpl) t).span;
+    public static co.elastic.apm.agent.impl.transaction.TransactionImpl accessTransaction(Transaction t) {
+        return (co.elastic.apm.agent.impl.transaction.TransactionImpl) ((TransactionImpl) t).span;
     }
```

Comment:
```suggestion
    public static TransactionImpl accessTransaction(Transaction t) {
        return (TransactionImpl) ((TransactionImpl) t).span;
    }
```

Reply:
Those two are not the same `TransactionImpl`. It's `co.elastic.apm.agent.impl.transaction.TransactionImpl` and `co.elastic.apm.api.TransactionImpl`.